frontend/src/main/java/com/google/tripmeout/frontend/storage/PlaceVisitStorage.java

```diff
@@ -17,30 +17,30 @@
    * @param placeVisit the PlaceVisitModel object to add to storage
    *
    * @throws a PlaceVisitAlreadyExists exception if there is a PlaceVisitModel
-   *     in storage with the same tripId and placeId as placeVisit
+   *     in storage with the same tripId and placeUuid as placeVisit
    */
   void addPlaceVisit(PlaceVisitModel placeVisit) throws PlaceVisitAlreadyExistsException;
 
   /**
-   * removes from storage the PlaceVisitModel whose tripId and placeId match the
-   * given tripId and placeId, respectively
+   * removes from storage the PlaceVisitModel whose tripId and placeUuid match the
+   * given tripId and placeUuid, respectively
    *
-   * @param tripId the id to match PlaceVisitModel object's placeId field on
-   * @param placeId the id to match PlaceVisitModel object's placeId field on
+   * @param tripId the id to match PlaceVisitModel object's tripId field on
+   * @param placeUuid the id to match PlaceVisitModel object's uuid field on
    *
    * @throws a PlaceVisitNotFound exception if there is no PlaceVisitModel
-   *     object in storage with the given tripId and placeId
+   *     object in storage with the given tripId and placeUuid
    */
-  void removePlaceVisit(String tripId, String placeId) throws PlaceVisitNotFoundException;
+  void removePlaceVisit(String tripId, String placeUuid) throws PlaceVisitNotFoundException;
```

Comment:
Nit: it would be nice to update these parameter names as well (and the ones in the in-memory implementation), though not critical.